content_handler/vulkan_rasterizer.cc

```diff
@@ -109,7 +103,7 @@ sk_sp<SkSurface> VulkanRasterizer::VulkanSurfaceProducer::ProduceSurface(
   auto sk_surface = SkSurface::MakeFromBackendRenderTarget(
       context_.get(), desc, SkColorSpace::MakeSRGB(), &props);
   if (!sk_surface) {
-    FTL_DLOG(INFO) << "MakeFromBackendRenderTarget Failed";
+    FTL_LOG(ERROR) << "MakeFromBackendRenderTarget Failed";
```

Comment:
I did this change because I wasn't seeing any of the error messages in release mode, which was making it hard to debug the race tracked by SY-88. If theres a problem with using FTL_LOG(ERROR) then I certainly don't mind changing it back, but its FTL_LOG(ERROR) seems semantically more in line with what I'm trying to express here than FTL_DLOG(INFO) (i.e. these things should not happen under normal circumstances, and even in release mode they indicate a real problem that needs to be addressed). When SY-88 is resolved there will be some way that we check if a graphics driver is available, and if there is then any failure following that point will be a legit error, and if not then we will fall back on the software backend without logging an error.